src/main/controllers/helpers/JudgmentHelpers.ts

```diff
@@ -209,217 +139,68 @@ export const getDecisionDetails = (
-      },
-      value: {
-        text: selectedDecisionApplication?.value.applicant,
-      },
-    },
-    {
-      key: {
-        text: translations.applicationType,
-        classes: 'govuk-!-font-weight-regular-m',
-      },
-      value: {
-        text: selectedDecisionApplication?.value.type,
-      },
-    },
-    {
-      key: {
-        text: translations.applicationDate,
-        classes: 'govuk-!-font-weight-regular-m',
-      },
-      value: {
-        text: selectedDecisionApplication?.value.date,
-      },
-    },
-    {
-      key: {
-        text: translations.legend,
-        classes: 'govuk-!-font-weight-regular-m',
-      },
-      value: {
-        text: selectedDecisionApplication?.value.details,
-      },
-    }
+    addSummaryRow(translations.applicant, selectedDecisionApplication?.value.applicant),
+    addSummaryRow(translations.applicationType, selectedDecisionApplication?.value.type),
+    addSummaryRow(translations.applicationDate, selectedDecisionApplication?.value.date),
+    addSummaryRow(translations.legend, selectedDecisionApplication?.value.details)
   );
 
   if (selectedDecisionApplication?.value.documentUpload) {
-    applicationDetails.push({
-      key: {
-        text: translations.supportingMaterial,
-        classes: 'govuk-!-font-weight-regular-m',
-      },
-      value: {
-        html: selectedApplicationDocDownloadLink,
-      },
-    });
+    applicationDetails.push(addSummaryHtmlRow(translations.supportingMaterial, selectedApplicationDocDownloadLink));
   }
-  applicationDetails.push({
-    key: {
-      text: translations.copyCorrespondence,
-      classes: 'govuk-!-font-weight-regular-m',
-    },
-    value: {
-      text: selectedDecisionApplication?.value.copyToOtherPartyYesOrNo,
-    },
-  });
+
+  applicationDetails.push(
+    addSummaryRow(translations.copyCorrespondence, selectedDecisionApplication?.value.copyToOtherPartyYesOrNo)
+  );
 
   if (selectedDecisionApplication?.value.respondCollection) {
     responseDetails.push(
-      {
-        key: {
-          text: translations.responseFrom,
-          classes: 'govuk-!-font-weight-regular-m',
-        },
-        value: {
-          text: selectedDecisionApplication?.value.respondCollection[0].value.from,
-        },
-      },
-      {
-        key: {
-          text: translations.date,
-          classes: 'govuk-!-font-weight-regular-m',
-        },
-        value: {
-          html: selectedDecisionApplication?.value.respondCollection[0].value.date,
-        },
-      },
-      {
-        key: {
-          text: translations.responsePart1 + responseFrom + translations.responsePart2,
-          classes: 'govuk-!-font-weight-regular-m',
-        },
-        value: {
-          html: selectedDecisionApplication?.value.respondCollection[0].value.response,
-        },
-      }
+      addSummaryRow(translations.responseFrom, selectedDecisionApplication?.value.respondCollection[0].value.from),
```

Comment:
referencing `selectedDecisionApplication?.value.respondCollection[0].value` more than 3 times, can we extract this out please?